zenoh-flow/src/model/dataflow/record.rs

```diff
@@ -336,7 +323,55 @@ impl TryFrom<(DataFlowDescriptor, Uuid)> for DataFlowRecord {
             }
         }
 
+        // A Loop between two nodes is, in fact, a backward link.
+        //
+        // To add a link we first need to add an input to the Ingress and an output to the Egress
+        // (as these ports were not declared by the user). We also must perform these additions to
+        // the `validator` as it will check that everything is alright on the link we are about to
+        // create.
+        //
+        // We finally add the loop information to the Ingress and Egress.
+        if let Some(loops) = d.loops {
+            for ciclo in loops {
```

Comment:
Add comment like: `// ciclo is the Italian word for loop, the latter cannot be used because it's keyword`